data/transactions/logic/blackbox_test.go

```diff
@@ -80,7 +80,7 @@ func TestNewAppEvalParams(t *testing.T) {
 	for i, param := range params {
 		for j, testCase := range cases {
 			t.Run(fmt.Sprintf("i=%d,j=%d", i, j), func(t *testing.T) {
-				ep := logic.NewAppEvalParams(testCase.group, &param, nil, 0)
+				ep := logic.NewAppEvalParams(testCase.group, &param, nil)
```

Comment:
Removes use of an extra parameter unneeded by this new way to track the created resources.